### PR TITLE
migrated remaining package.xml files to format=2

### PR DESCRIPTION
**Pull request by Fern-and-dough**

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
This is for https://github.com/ros-industrial/ros_industrial_issues/issues/56, correct?

**Comment by Fern-and-dough** [comment]
Yes, this is for ros-industrial/ros_industrial_issues#56, apologies for not being descriptive.

**Comment by gavanderhoorn** [comment]
No problem. At least we keep things connected like this.

**Comment by gavanderhoorn** [comment]
Travis is going to take a while unfortunately.

File: abb_irb1200_support/package.xml [hunk 1]
```diff
@@ -36,11 +36,11 @@
 
   <build_depend>roslaunch</build_depend>
```
Comment (trentwall) [inline abb_irb1200_support/package.xml]: <build_depend>roslaunch</build_depend> should be replaced with  <depend>roslaunch</depend>

File: abb_irb4600_support/package.xml [hunk 2]
```diff
@@ -37,12 +37,12 @@
 
   <build_depend>roslaunch</build_depend>
```
Comment (trentwall) [inline abb_irb4600_support/package.xml]: <build_depend>roslaunch</build_depend> should be replaced with   <depend>roslaunch</depend>

File: abb_irb52_support/package.xml [hunk 3]
```diff
@@ -34,14 +34,15 @@
   <url type="repository">https://github.com/ros-industrial/abb_experimental</url>
 
   <buildtool_depend>catkin</buildtool_depend>
+  
   <build_depend>roslaunch</build_depend>
```
Comment (trentwall) [inline abb_irb52_support/package.xml]: <build_depend>roslaunch</build_depend> should be replaced with   <depend>roslaunch</depend>

File: abb_irb6650s_support/package.xml [hunk 4]
```diff
@@ -34,10 +35,10 @@
   <buildtool_depend>catkin</buildtool_depend>
   <build_depend>roslaunch</build_depend>
```
Comment (trentwall) [inline abb_irb6650s_support/package.xml]: <build_depend>roslaunch</build_depend> should be replace   <depend>roslaunch</depend>

File: abb_irb7600_support/package.xml [hunk 5]
```diff
@@ -37,12 +37,12 @@
 
   <build_depend>roslaunch</build_depend>
```
Comment (trentwall) [inline abb_irb7600_support/package.xml]: <build_depend>roslaunch</build_depend> should be replaced with   <depend>roslaunch</depend>

File: abb_irb4600_support/package.xml (hunk 2 above)
Comment (trentwall) [reply abb_irb4600_support/package.xml]: <build_depend>roslaunch</build_depend> should be relplaced with   <depend>roslaunch</depend>

**Review by trentwall — CHANGES_REQUESTED** [review]

**Comment by gavanderhoorn** [comment]
@trentwall: what do you believe the `build_depend`s on `roslaunch` should be?

In the `fanuc` repository we've made them `test_depend` (as that is what it's used for): [here](https://github.com/ros-industrial/fanuc/blob/7fc71ef83e8a6f7c8ad2c17ea6f5933d69965142/fanuc_cr7ia_support/package.xml#L43).

File: abb_irb4600_support/package.xml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<package>
+<package="2">
```
Comment (jrgnicho) [inline abb_irb4600_support/package.xml]: I believe the syntax is `<package format="2">`

http://docs.ros.org/jade/api/catkin/html/howto/format2/migrating_from_format_1.html#migrating-from-format1-to-format2

File: abb_irb52_support/package.xml [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<package>
+<package="2">
```
Comment (jrgnicho) [inline abb_irb52_support/package.xml]: Same as above

File: abb_irb6650s_support/package.xml [hunk 8]
```diff
@@ -1,4 +1,5 @@
-<package>
+<?xml version="1.0" ?>
+<package="2">
```
Comment (jrgnicho) [inline abb_irb6650s_support/package.xml]: Same as above

**Review by jrgnicho — CHANGES_REQUESTED** [review]
Modify to use the correct syntax as described in http://docs.ros.org/jade/api/catkin/html/howto/format2/migrating_from_format_1.html#migrating-from-format1-to-format2

File: abb_irb1200_support/package.xml (hunk 1 above)
Comment (JeremyZoss) [reply abb_irb1200_support/package.xml]: replaced with `<test_depend>roslaunch</test_depend>`, as suggested by @gavanderhoorn

File: abb_irb4600_support/package.xml (hunk 2 above)
Comment (JeremyZoss) [reply abb_irb4600_support/package.xml]: replaced with `<test_depend>roslaunch</test_depend>`, as suggested by @gavanderhoorn

File: abb_irb52_support/package.xml (hunk 3 above)
Comment (JeremyZoss) [reply abb_irb52_support/package.xml]: replaced with `<test_depend>roslaunch</test_depend>`, as suggested by @gavanderhoorn

File: abb_irb6650s_support/package.xml (hunk 4 above)
Comment (JeremyZoss) [reply abb_irb6650s_support/package.xml]: replaced with `<test_depend>roslaunch</test_depend>`, as suggested by @gavanderhoorn

File: abb_irb7600_support/package.xml (hunk 5 above)
Comment (JeremyZoss) [reply abb_irb7600_support/package.xml]: replaced with `<test_depend>roslaunch</test_depend>`, as suggested by @gavanderhoorn

File: abb_irb4600_support/package.xml (hunk 6 above)
Comment (JeremyZoss) [reply abb_irb4600_support/package.xml]: agreed.  fixed in commit below.

File: abb_irb52_support/package.xml (hunk 7 above)
Comment (JeremyZoss) [reply abb_irb52_support/package.xml]: agreed.  fixed in commit below.

File: abb_irb6650s_support/package.xml (hunk 8 above)
Comment (JeremyZoss) [reply abb_irb6650s_support/package.xml]: agreed.  fixed in commit below.

**Review by jrgnicho — APPROVED** [review]

**Review by JeremyZoss — APPROVED** [review]
Thanks @gavanderhoorn and @jrgnicho .  Fixed per your comments.  Looks good to me, no build issues.

**Pull request merged**